crates/katana-rpc/src/lib.rs

```diff
@@ -35,8 +37,18 @@ where
         let mut methods = KatanaRpc::new(self.sequencer.clone()).into_rpc();
         methods.merge(StarknetRpc::new(self.sequencer.clone()).into_rpc())?;
 
+        let cors = CorsLayer::new()
+            // Allow `POST` when accessing the resource
+            .allow_methods([Method::POST])
+            // Allow requests from any origin
+            .allow_origin(Any)
```

Comment:
Can we also add an issue to make this configurable?

Reply:
yes good point - ill do that next